Podcast/UnderlineTabBarView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  UnderlineTabBarView.swift
+//  Eatery
+//
+//  Created by Eric Appel on 11/4/15.
+//  Copyright © 2015 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol TabBarDelegate: class {
+    func selectedTabDidChange(_ newIndex: Int)
```

Comment:
selectedTabDidChangeToNewIndex(_ newIndex: Int) or whatever you chose for the earlier comment